### PR TITLE
fix: skip codecov publish report step in ci pipeline

### PR DESCRIPTION
**Pull request by v-rakeshsh**
#### Details

Skip codecoverage step in pipeline as codecov package is deprecated.

#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [x] Ran `yarn fastpass`
- [ ] Added/updated relevant unit test(s) (and ran `yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [ ] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -79,33 +79,33 @@ jobs:
...
+  #     timeout-minutes: 2
+
+  #   - uses: actions/download-artifact@v4
+  #     with:
+  #       name: unit-tests-2-results
+  #       path: unit-tests-2-results
+  #     timeout-minutes: 2
+
+  #   - run: npx codecov
+  #     timeout-minutes: 3
```
Comment (brocktaylor7) [inline .github/workflows/ci.yml:108]: Let's delete these lines rather than comment them out.

File: pipeline/build-shared.yaml [hunk 2]
```diff
@@ -90,9 +90,9 @@ extends:
                           displayName: publish code coverage
                           timeoutInMinutes: 5
 
-                        - script: yarn publish-code-coverage -t $(CODECOV_TOKEN)
-                          displayName: Publish code coverage to codecov
-                          timeoutInMinutes: 3
+                        # - script: yarn publish-code-coverage -t $(CODECOV_TOKEN)
+                        #   displayName: Publish code coverage to codecov
+                        #   timeoutInMinutes: 3
```
Comment (brocktaylor7) [inline pipeline/build-shared.yaml:95]: Same here, let's delete these.

**Review by brocktaylor7 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (v-rakeshsh) [reply .github/workflows/ci.yml]: Hi @brocktaylor7 removed now, please re-review.

File: pipeline/build-shared.yaml (hunk 2 above)
Comment (v-rakeshsh) [reply pipeline/build-shared.yaml]: Hi @brocktaylor7 removed now, please re-review.

**Review by madalynrose — APPROVED** [review]

**Pull request merged**